commitlint.config.ts

```diff
@@ -68,6 +68,10 @@ function includeHashtagRef(text: string) {
     return text.match(`#[0-9]+`) !== null;
 }
 
+function removeCodeBlock(text: string) {
```

Comment:
@realmarv does this function only remove one code block? or does it remove any code blocks? (if the former, then it's wrong; if the latter, then its name is wrong)

Reply:
It's the latter. How about removeCodeBlocks or removeAllCodeBlocks?

Reply:
removeAllCodeBlocks :+1:

Reply:
Done